src/stdlib/list.ts

```diff
@@ -14,7 +14,7 @@ declare global {
 // Author: Martin Henz
 // Translated to TypeScript by Evan Sebastian
 
-export type List = Value[]
+export type List = Value[] | null
```

Comment:
This shouldn't be changed. Would imply that stuff like `head(null)` should work, but it actually throws an error. Tbh half the type annotations don't really make sense anyway since we are working off SICP, loads of stuff go straight to `any` type haha.

If you really want to, make `is_pair` a [type guard] so that you don't need to spam `xs!` everywhere.

[type guard]: https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types

Reply:
Yup. I've changed it so that some functions like map return `List | null` instead. But it still looks quite ugly.